doc/articles/uno-development/building-uno-ui.md

```diff
@@ -50,6 +50,7 @@ The step by step process is:
 1. To confirm that everything works:
    - For iOS/Android/macOS you can right-click on the `Uno.UI` project in the Solution Explorer and 'Build'.
    - For WebAssembly and Skia you can right-click on the `Uno.UI.Runtime.WebAssembly` or `Uno.UI.Runtime.Skia.[Gtk|Wpf]` project in the Solution Explorer and 'Build'.
+1. Optionally adjust additional parameters in `crosstargeting_override.props`, such as `UnoDisableNetAnalyzers` which can improve the build time during debugging sessions.
```

Comment:
VS disables all analyzers when for builds initiated by "Debug tests" or "Run with debugger" already. So analyzers don't run unless an explicit build (Ctrl+Shift+B or right click project then Build).

So the comment is quite misleading IMO.

Reply:
@jeromelaban @mrlacey Can I know more about the performance problem being referred to here? A binlog will also help to properly report the specific slow analyzer to Microsoft.

Reply:
I revised the logic responsible for that in Roslyn, and build optimization when building for debugging is only done when `TreatWarningsAsErrors` isn't true.

To force build optimization, `OptimizeImplicitlyTriggeredBuild` should be set to true.

I think the better fix is to set `OptimizeImplicitlyTriggeredBuild` to true in `Directory.Build.props` (just below `TreatWarningsAsErrors`). This way, all developers will have the same experience, and they get it "by default" (without having to modify `crosstargeting_override.props`).

Reply:
After adding `<OptimizeImplicitlyTriggeredBuild>true</OptimizeImplicitlyTriggeredBuild>`, you should see the following message if the build is initiated by "Debug test" or similar:

> Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.

Reply:
Interesting, then maybe we should set the property instead :) Thanks for the insights!

Reply:
@jeromelaban I'd set it unconditionally in Directory.Build.props

Reply:
Let's have it in the override file for now, and if it does not cause local build issues (or CI issues) we can revisit.